core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -618,6 +618,7 @@ write_files:
       ETCD_ADVERTISE_CLIENT_URLS=https://$advertised_hostname:2379
       ETCD_LISTEN_PEER_URLS=https://$private_ip:2380
       ETCD_INITIAL_ADVERTISE_PEER_URLS=https://$advertised_hostname:2380" >> /var/run/coreos/etcd-environment
+      ETCD_VERSION={{.Etcd.Version}}
```

Comment:
there is an easy to miss redirect happening in this file, line you add should be inside that redirect to have an effect